components/home/news.html

```diff
@@ -1,30 +1,35 @@
 <div class="container">
```

Comment:
Might want to rename this file, three-fs.html?

Reply:
Yep. Good call.